docs/user-guide/deploy-chart/chart-group.md

```diff
@@ -7,39 +7,35 @@ Create Chart Group And Deploy App store applications.
 
 ## 1. Create Group 
 
-Select the `Charts` section from the left pane, you will be landed to the `Chart Store` page. 
-Click on `Discover` and Click Button on right top `Create Group`.
+Select the `Charts` from the left pane, you will be landed to the `Chart Store` page. 
```

Comment:
Select `Charts` from the left pane, you will reach to the `Chart Store` page.

Reply:
done